classes/03class/exercises/c03-docker01/carloshz4/README.md

```diff
@@ -0,0 +1,26 @@
+# C03-Docker01
+
+## Docker 
+- [Dockerfile](Dockerfile)
+
+## Command Execution Output
+- Command to build the image:
+```
+docker build -t da-terraform:c03-docker-e01 .
+
+docker images |egrep "REPOSITORY|da-terraform"
```

Comment:
Another way is to use [`--filter` option](https://docs.docker.com/engine/reference/commandline/images/#filtering) like: `docker images --filter reference=da-terraform`